video/image_writer.c

```diff
@@ -52,6 +56,15 @@ const struct image_writer_opts image_writer_opts_defaults = {
     .webp_compression = 4,
     .jxl_distance = 1.0,
     .jxl_effort = 4,
+    .avif_encoder = "libaom-av1",
+    .avif_pixfmt = "yuv420p",
```

Comment:
How does this work if there's alpha? Hardware screenshots contain alpha by default.